### PR TITLE
Replace all mentions of the matrix channel with links to zulip

### PR DESCRIPTION
**Pull request by xs5871**
noqa on the portuguese and japanese.

### CODE REVIEW

**Comment by klardotsh** [comment]
Not sure if @razelsantos280 and @AndersonTorres are still around to review JP and PTBR, respectively, but I'll see if they're able to jump in. If not I'll merge this late weekend / early in the coming week.

File: docs/ptBR/Getting_Started.md [hunk 1]
```diff
@@ -123,8 +123,6 @@ usuários](https://github.com/KMKfw/user_keymaps) que fornecemos e fuce nossa
 Caso precise, ajuda para depuração pode ser encontrada [aqui](debugging.md).
 
 Se você precisa de suporte com o KMK ou quer somente dizer oi, encontre-nos no
-canal [#kmkfw:klar.sh no Matrix](https://matrix.to/#/#kmkfw:klar.sh). Este canal
-tem uma ponte no Discord
-[aqui](https://discordapp.com/widget?id=493256121075761173&theme=dark) por
-conveniência. Se você precisa de ajuda ou pretende abrir um bug report, se
+canal [no Zulip](https://kmkfw.zulipchat.com).
```
Comment (AndersonTorres) [inline docs/ptBR/Getting_Started.md:126]:
```suggestion
canal [kmkfw no Zulip](https://kmkfw.zulipchat.com).
```

**Review by AndersonTorres — APPROVED** [review]
Besides, LGTM.

**Comment by xs5871** [comment]
> Not sure if @razelsantos280 and @AndersonTorres are still around to review JP and PTBR

That is valuable information, thanks.

**Pull request merged**